src/FSharpVSPowerTools.Logic/QuickInfoMargin.fs

```diff
@@ -136,7 +136,7 @@ type QuickInfoMargin (textDocument: ITextDocument,
             |> Async.StartInThreadPoolSafe
         | None -> updateQuickInfo (None, None)
 
-    let docEventListener = new DocumentEventListener ([ViewChange.layoutEvent view; ViewChange.caretEvent view], 1000us, updateAtCaretPosition)
+    let docEventListener = new DocumentEventListener ([ViewChange.layoutEvent view; ViewChange.caretEvent view], 200us, updateAtCaretPosition)
```

Comment:
Ah, I see.  I'm mostly concerned about flooding FCS when there are lots of rapid edits.  FCS will cache one ParseAndCheckFileInProject request for each file, keyed on the source text.  If the source text is changing, this cache is naturally of no use.  So asyncs that flood FCS with requests with different source texts will quickly create too much work.

One way to help here is to cancel outstanding asyncs relating to a previous source text once an edit has been made.  Another is to put in a delay before starting the request (the delay is in itself a form of cancellation - it's giving a pause to allow a cancellation to come through).  The next FCS nuget will include the ability to cancel requests, see [this](https://github.com/fsharp/FSharp.Compiler.Service/pull/445/files#diff-d784b269182ed2ba106aaa72b68ef9dfR48)